apps/studio/src/components/connection/CommonEntraId.vue

```diff
@@ -0,0 +1,222 @@
+        <label for="msiEndpoint">MSI Endpoint</label>
+        <input
+          name="msiEndpoint"
+          type="text"
+          class="form-control"
+          v-model="config.azureAuthOptions.msiEndpoint"
+        >
+      </div>
+    </div>
+  </div>
+
+</template>
+<script>
+import { AzureAuthType } from "@/lib/db/types";
+import { AppEvent } from "@/common/AppEvent";
+import _ from "lodash";
+import { onMounted } from "vue";
+
+export default {
+  props: ['config', 'authType'],
+  data() {
+    return {
+      azureAuthEnabled: this.config.azureAuthOptions?.azureAuthEnabled,
+      accountName: null,
+      signingOut: false,
+      errorSigningOut: null,
+    };
+  },
+  computed: {
+    username: {
+      get() {
+        return this.config.username || this.config.user;
+      },
+      set(value) {
+        this.config.username = value;
+        this.config.user = value;
+      }
+    },
+    showUser() {
+      return [AzureAuthType.Password, AzureAuthType.CLI].includes(this.authType)
+    },
+    showPassword() {
+      return [AzureAuthType.Password].includes(this.authType)
+    },
+    showTenantId() {
+      return [AzureAuthType.Password, AzureAuthType.ServicePrincipalSecret].includes(this.authType)
+    },
+    showClientSecret() {
+      return [AzureAuthType.ServicePrincipalSecret].includes(this.authType)
+    },
+    showMsiEndpoint() {
+      return [AzureAuthType.MSIVM].includes(this.authType)
+    },
+    showCli() {
+      return [AzureAuthType.CLI].includes(this.authType)
+    },
+    hasAccessTokenCache() {
+      return Boolean(this.accountName)
+    },
+  },
+  watch: {
+    async authType() {
+      const authId = this.config.azureAuthOptions?.authId || this.config?.authId
+      if (this.authType === AzureAuthType.AccessToken && !_.isNil(authId)) {
+        this.accountName = await this.connection.azureGetAccountName(authId);
+      } else {
+        this.accountName = null
+      }
+    },
+    azureAuthEnabled() {
+      this.config.azureAuthOptions.azureAuthEnabled = this.azureAuthEnabled
+    }
+  },
+  methods: {
+    toggleIAMAuthentication() {
+      this.azureAuthEnabled = !this.azureAuthEnabled
+      this.config.azureAuthOptions.azureAuthEnabled = this.azureAuthEnabled;
+    },
+    async signOut() {
+      try {
+        this.signingOut = true
+        await this.connection.azureSignOut(this.config);
+        this.config.authId = null
+        this.accountName = null
+      } catch (e) {
+        this.errorSigningOut = e
+        this.$emit('error', e)
+      } finally {
+        this.signingOut = false
+      }
+    },
+    async tryFindAzCli() {
+      if (!this.config.azureAuthOptions.cliPath) {
+        try {
+          const result = await window.main.rawInvoke('which-tool', 'az')
+          if (result.success) {
+            this.config.azureAuthOptions.cliPath = result.path;
+          } else {
+            this.config.azureAuthOptions.cliPath = 'NO CLI FOUND, PLEASE REFER TO DOCS'
```

Comment:
This shouldn't be assigned to the `cliPath` variable, have a `cliError` variable on the component and show that to the user when you can't find the path.

Reply:
Tell them they need to set the path explicitly in their settings file, then link directly to that page on how to do it.